src/CattyTests/Utils/UtilTests.swift

```diff
@@ -152,4 +162,37 @@ final class UtilTests: XCTestCase {
         let devicePlatformVersion = UIDevice.current.systemVersion
         XCTAssertEqual(utilPlatformVersion, devicePlatformVersion)
     }
+
+    func testAllMessagesForProjectIsEmptyAtInit() {
+
+        let messages = Util.allMessages(for: project)
+
+        XCTAssertEqual(messages?.count, 0)
+    }
+
+    func testAllMessagesForProjectWithValues() {
+
+        project.allBroadcastMessages?.add("firstValue")
+
+        spriteObject = SpriteObject()
+        project.objectList.add(spriteObject!)
+        broadcastScript = BroadcastScript()
+        spriteObject.scriptList.add(broadcastScript!)
+        broadcastScript.receivedMessage = "secondValue"
+
+        script = Script()
+        spriteObject.scriptList.add(script!)
+
+        broadcastBrick = BroadcastBrick()
+        script.brickList.add(broadcastBrick!)
+        broadcastBrick.broadcastMessage = "thirdValue"
+
+        broadcastWaitBrick = BroadcastWaitBrick()
+        script.brickList.add(broadcastWaitBrick!)
+        broadcastWaitBrick.broadcastMessage = "fourthValue"
+
+        let messages = Util.allMessages(for: project)
+
+        XCTAssertEqual(messages?.count, 4)
+    }
```

Comment:
Please add an additional test case verifying that two same entries added to `allBroadcastMessages` will only return one unique result (without duplicates).

Reply:
Done :D